packages/webpack-model/src/modules-to-foam-tree.ts

```diff
@@ -145,12 +145,6 @@ export default function modulesToFoamTree(
 
   for (const module of modules) {
     handleModule(root, module, getModuleSize);
-
-    if (module.modules) {
```

Comment:
This will hide all inner modules of concat modules
The better solution is to ignore root concat module size for calculation